issue_15/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js" defer></script>
+    <title>もりけん塾 JS課題15</title>
+</head>
+
+<body>
+    <div class="btn__wrapper show-btn__wrapper">
+        <button type="button" class="btn show-btn" id="js-show-btn">モーダルを表示する</button>
+    </div>
+    <div class="modal" id="js-modal">
+        <form action="" method="post" id="js-form">
+            <div class="modal__element">
+                <label for="js-input-number" class="modal__input-label">数字</label>
+                <input type="number" name="num" class="modal__input" id="js-input-number">
```

Comment:
IMO
Maybe 'number' is better??
```suggestion
                <input type="number" name="number" class="modal__input" id="js-input-number">
```

Reply:
https://github.com/yoshikawa-akr/js-lessons/pull/15#discussion_r990944090